src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -76,7 +76,22 @@ Sentry does not know if a local variable that looks like a credit card number ac
 
 ## Selectors
 
-Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by variable/field name from the event, but can also be used to conservatively test rules on real data.
+Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by variable/field name from the event, but can also be used to conservatively test rules on real data. A few examples:
+
+* `**` to scrub everywhere
+* `$error.value` to scrub in the exception message
+* `$message` to scrub the event-level log message
+* `extra.'My Value'` to scrub the key `My Value` in "Additional Data"
```

Comment:
Should we call out that the quotes are required to group together custom fields that have spaces or other special characters? (If that is actually the case)

Reply:
We do this further down (outside of this diff). This is only supposed to be a cheatsheet of some sort.

Reply:
Found it, sorry I should have looked at the whole file.

[#escaping-special-characters](https://github.com/getsentry/sentry-docs/blob/aeb3af72cf77460990506cc2e969677fcfb57fa1/src/collections/_documentation/data-management/advanced-datascrubbing.md#escaping-special-characters)

In some cases, like in the Breadcrumbs payload docs, we add intra-page links. In this case it might not be worth it, would probably mean the description here gets longer to include a descriptive link.